guides/slack.md

```diff
@@ -0,0 +1,67 @@
+
+ - Project channels (named after the project). If you are working on that delivery, it is expected you'll be in that channel, otherwise open project channels are optional.
+ - Supply channels (prefixed `supply-`). These channels are used by groups focusing on how we _supply_ deliverables. Optional unless directly involved with this group.
+ - Demand channels (prefixed `demand-`). These channels are used by groups focusing on how we engage with other organisations. Optional unless directly involved with this group.
+ - Group channels (such as `ops-` or `academy-`). Channels used by 
+ - Specific purpose channels (usually prefixed with their temporary purpose, for example `bid-`). Temporary channels used for teams fulfilling a specific related task. Optional unless involved with the task, and will be archived after completion.
+ - General channels (no prefix). Company wide chanels such as `announcements`, `help` or regional channels such as `manchester` and _community of practice_ channels prefixed with `cop-`, these channels are used for cross company communication and tend have channel specific usage expecations, which should be detailed in their channel topic.
+ - Off topic channels (prefixed `x-`). These channels are entirely optional to join and use, feel free to create off topic channels if your interest or hobby is not yet represented!
+ - Bot channels or 'firehose' channels. It is occasionally useful to setup bots that post information to specific channels periodically, it is advised that you only need to join these channels if you are involved in monitoring such information.
+ 
+## Mandatory channels
+
+As of writing, it is expected every slack user will be present in `#announcements`, this is so that everyone can be reached in case of an important announcement.
+There are no other channels mandatory for _all_ users, but it is expected you will be in channels relevant to your working group.
+
+## Private channels and group chats
+
+Private channels can be used for specific tasks and group chats may be used to have small private discussions if necessary. It's worth noting that everything posted on our slack is logged and could be subject to a Freedom of Information request (specifically for channels relating to public sector projects), so it is wise to always use slack as if your message could be read.
+ 
+# Notifications and Threads
+ 
+It's generally expected you'll be available in relevant, non-optional channels during working hours - this means project/group channels you're directly involved in and `#announcements`. When minimising notification noise, it's entirely reasonable for you to set availability with the teams you're working with. All other optional channels are exactly that, optional. It is also reasonable to mute slack entirely and check periodically, whatever works best for you, remembering that slack is the primary method of communication within the company while remote.
```

Comment:
I'm wary of saying "you should be available" this broadly - I would expect some people to interpret this as 
> you need to have slack notifications on,  and immediately reply to them, or you're in trouble

... and I don't think that's a good norm!

I know the follow-up sentences go into this, but people will often skim rather than fully read guides like this, and might take away the wrong lesson.

Reply:
There's nuance to this, and skimming it will make a reader miss that nuance regardless of what we write. You _should_ be available and what that looks like is up to you to develop an understanding of with your team. We could condense it to that one sentence, but I like clarifying that it's acceptable to mute and focus.

I think this is better covered in the onboarding so will look at rewording in a similar way.

Reply:
We could flag that there's that tension in the opening sentence(s)? Can't get the wording right, but something like:
> Your team will expect you to read relevant channels and respond to them where you're needed, but there's no rule for how quickly you need to respond. There's a tension between being 'available' and distracted by notifications. Talk with your team and develop a way of working that works for you (and them!).

Reply:
Yeah I like that, will take another look